release_notes/42-release-notes.adoc

```diff
@@ -91,7 +95,7 @@ Based on testing with a 50-cluster and 4900-node-environment, these improvement
 
 If you have a disconnected environment or use registry mirrors to satisfy your organizational controls on external content, {product-title-short} can now scan images from your registry mirrors in {ocp} environments.
 
-This feature leverages the *delegated image scanning* functionality introduced in {product-title-short} 4.1. The delegated image scanning functionality allows {product-title-short} Secured cluster services to use Sensor to pull images for scanning from registry mirrors you have set up the `ImageContentSourcePolicy`, `ImageDigestMirrorSet`, or `ImageTagMirrorSet` custom resources.
+This feature leverages the *delegated image scanning* functionality introduced in {product-title-short} 4.1. The delegated image scanning functionality allows {product-title-short} Secured cluster services to use Sensor to pull images for scanning from registry mirrors you have set up using the `ImageContentSourcePolicy`, `ImageDigestMirrorSet`, or `ImageTagMirrorSet` custom resources.
```

Comment:
```suggestion
This feature leverages the *delegated image scanning* functionality introduced in {product-title-short} 4.1. The delegated image scanning functionality allows {product-title-short} Secured cluster services to use Sensor to pull images for scanning from registry mirrors you have set up by using the `ImageContentSourcePolicy`, `ImageDigestMirrorSet`, or `ImageTagMirrorSet` custom resources.
```

Reply:
This is outside of the sciope of this PR, but can we delete the _introduced in {product-title-short} 4.1_ at this point?